build.install4j

```diff
@@ -334,6 +334,177 @@
+    if(System.getenv("XDG_DATA_HOME") != null || (shareFolder.exists() &amp;&amp; shareFolder.isDirectory())){
+        File applicationsDir = new File(shareFolder, "applications");
+        if(!applicationsDir.exists() || !applicationsDir.isDirectory()){
+            applicationsDir.mkdirs();
+        }
+        File desktopFile = Arrays.stream(applicationsDir.listFiles(new FilenameFilter(){
+            @Override
+            public boolean accept(File dir, String name){
+                return name.matches("TripleA[\\-0-9\\.]*\\.desktop");
+            }
+        })).findAny().get();
+        File mimeappsFile = new File(applicationsDir, "mimeapps.list");
+        if(!desktopFile.exists()){
+            return false;
+        }
+        if(!mimeappsFile.exists()){
+            mimeappsFile.createNewFile();
+        }
+        String mimeType = "x-scheme-handler/triplea";
+        StringBuilder builder = new StringBuilder();
+        try(Scanner scanner = new Scanner(desktopFile)){
+            String mimeTypePrefix = "MimeType=";
+            while(scanner.hasNextLine()){
+                String nextLine = scanner.nextLine();
+                if(nextLine.startsWith("Exec=") &amp;&amp; !nextLine.contains("%u")){
+                    builder.append(nextLine).append(" \"%u\"\n");
+                } else if(!nextLine.startsWith(mimeTypePrefix)){
+                    builder.append(nextLine).append('\n');
+                }
+            }
+            builder.append(mimeTypePrefix).append(mimeType);
+        }
+        try(PrintWriter writer = new PrintWriter(desktopFile)){
+            writer.println(builder.toString());
+        }
+        builder = new StringBuilder();
+        try(Scanner scanner = new Scanner(mimeappsFile)){
+            boolean isSetup = false;
+            String defaultApplicationString = "[Default Applications]";
+            String mimeTypePrefix = mimeType + "=";
+            while(scanner.hasNextLine()){
+                String nextLine = scanner.nextLine();
+                if(!nextLine.startsWith(mimeTypePrefix)){
+                    builder.append(nextLine).append('\n');
+                }
+                if(nextLine.equalsIgnoreCase(defaultApplicationString)){
+                    isSetup = true;
+                }
+            }
+            if(!isSetup){
+                builder.insert(0, defaultApplicationString + "\n");
+            }
+            builder.append(mimeTypePrefix).append(desktopFile.getName());
+        }
+        try(PrintWriter writer = new PrintWriter(mimeappsFile)){
+            writer.println(builder.toString());
+        }
+    }
+}
+return true;</string>
+                              </void>
+                            </object>
+                          </void>
+                        </object>
+                      </java>
+                    </serializedBean>
+                    <condition />
+                  </action>
+                </beans>
+              </group>
```

Comment:
Good point.
While currently on Windows old handlers are overridden properly (I just noticed this may not always be working on linux)
Handlers are completely when uninstalling any version.
I will fix that. Shouldn't be too difficult.